### PR TITLE
Enable Nullable

### PR DESCRIPTION
**Pull request by stefannikolei**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
Enabled nullable analysis. 
Disabled in all files where errors happened.

First step of #2231

### CODE REVIEW

**Comment by JimBobSquarePants** [comment]
Wow! Please tell me you did some sort of find/replace for this?

**Comment by stefannikolei** [comment]
> Wow! Please tell me you did some sort of find/replace for this?

Adding it in all files works by find and replace. 
Adding it only to files which had errors enforced me to use a good bottle of wine 😅

File: src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/Generated/Abgr32.PixelOperations.Generated.cs
Comment (JimBobSquarePants) [inline src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/Generated/Abgr32.PixelOperations.Generated.cs:60]: Any idea what caused these changes? 

We should probably add #nullable disable to all T4 templates at some point.

Comment (stefannikolei) [reply src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/Generated/Abgr32.PixelOperations.Generated.cs:60]: No clue. I reverted a bunch of changes. Probably overlooked this one. 

I can only imagine that opening the file in rider on Mac caused this.

**Review by JimBobSquarePants — APPROVED** [review]
This looks good to me. Thanks!

**Comment by JimBobSquarePants** [comment]
I broke your PR. Sorry!

**Comment by stefannikolei** [comment]
> I broke your PR. Sorry!

I fixed it. One build step is failing. I am not able to trigger it. For me the error seems unrelated. @JimBobSquarePants Could you have a look please?

**Comment by stefannikolei** [comment]
@JimBobSquarePants Is there anything blocking this pr? Anything I can do? Or you just waiting for a good "moment" to merge?

**Comment by JimBobSquarePants** [comment]
Sorry for the delay. Yep everything looks good. Just trying to get #2276 in first as it will be easier to merge this one after than the other way round.

**Comment by stefannikolei** [comment]
@JimBobSquarePants I added another nullable disable. So I should be finished now

**Review by JimBobSquarePants — APPROVED** [review]
Awesome! Thanks very much for doing this. I appreciate that it's a chore!

**Pull request merged**
